mlrun/projects/pipelines.py

```diff
@@ -354,11 +360,18 @@ def enrich_function_object(
         f.enrich_runtime_spec(
             project.spec.default_function_node_selector,
         )
-
+    skip_automount_for_kfp = False
+    if project.spec.workflows:
```

Comment:
Shouldn't this check if there's a `pipeline_context.workflow`? Looking here may still return a positive result even if just running a function and there's no workflow in the pipeline context.

Reply:
fixed